src/utils/app_names.py

```diff
@@ -64,4 +64,27 @@ class MetricNames(StrEnum):
     "debt_ebitda",
 ]
 
-ANONYMIZABLE_METRICS = ["revenue", "revenue_per_employee", "gross_profit"]
+ANONYMIZABLE_METRICS = [
+    "revenue",
+    "revenue_per_employee",
+    "gross_profit",
+    "cost_of_goods",
+    "sales_marketing",
+    "research_development",
+    "general_admin",
+    "other_operating_expenses",
+    "ebitda",
+    "run_rate_revenue",
+    "customer_lifetime_value",
+    "customer_acquition_costs",
+    "customer_annual_value",
+    "losses_and_downgrades",
+    "upsells",
+    "new_bookings",
+    "cash_and_equivalents",
+    "long_term_debt",
+    "equity_invested",
+    "cash_flow_operations",
+]
+
+CALCULATED_ANONYMIZABLE_METRICS = ["revenue", "revenue_per_employee", "gross_profit"]
```

Comment:
You have these metrics two times; in this variable and in the ANONYMIZABLE_METRICS, if you really need this one you can extend ANONYMIZABLE_METRICS with these values instead.